### DISCUSSION TITLE
Updating the instance type for the Kubernetes node pool

### DISCUSSION THREAD

**Discussion by Mohit-3196** [ant-media/Ant-Media-Server#5404] [FAQs]
I created a Kubernetes cluster and while doing so, in my node pool I used 4 core CPU Optimized instances since it was for initial testing purposes and then installed AMS using Helm.
Now that my testing is over and I want to move this to production, I'm looking to upgrade the instance type to 16 cores CPU-optimized instances in the node pool.

Is it possible to upgrade the instance size without having to recreate/reinstalling and losing my current AMS configurations? If so, then how?

Thank you

**Comment by muratugureminoglu** [comment]
If MongoDB is not in a different pool or in a remote environment, it is not possible to make this change without taking a backup of MongoDB.
